.github/workflows/iroha2-dev-pr-title-commit-msg.yml

```diff
@@ -92,3 +92,70 @@ jobs:
       - name: None of the above
         if: steps.fix-match.outputs.match == '' && steps.refactor-match.outputs.match == '' && steps.feature-match.outputs.match == '' && steps.docs-match.outputs.match == '' && steps.ci-match.outputs.match == ''
         run: exit 1
+
+  commit-msg-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Feature
+        id: feature-match
+        continue-on-error: true
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^\[feature\] #\d+(, #\d+)*: .+$'
+          error: 'Wrong [feature] commits message'
+          excludeDescription: 'true'
+          excludeTitle: 'true'
+          checkAllCommitMessages: 'true'
+          accessToken: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Refactor
+        id: refactor-match
+        continue-on-error: true
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^\[refactor\]( #\d+(, #\d+)*)?: .+$'
+          error: 'Wrong [refactor] commits message'
+          excludeDescription: 'true'
+          excludeTitle: 'true'
+          checkAllCommitMessages: 'true'
+          accessToken: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Fix
+        id: fix-match
+        continue-on-error: true
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^\[fix\] #\d+(, #\d+)*: .+$'
+          error: 'Wrong [fix] commits message'
+          excludeDescription: 'true'
+          excludeTitle: 'true'
+          checkAllCommitMessages: 'true'
+          accessToken: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Documentation
+        id: docs-match
+        continue-on-error: true
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^\[documentation\]( #\d+(, #\d+)*)?: .+$'
+          error: 'Wrong [documentation] commits message'
+          excludeDescription: 'true'
+          excludeTitle: 'true'
+          checkAllCommitMessages: 'true'
+          accessToken: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: CI
+        id: ci-match
+        continue-on-error: true
+        uses: gsactions/commit-message-checker@v1
+        with:
+          pattern: '^\[ci\]( #\d+(, #\d+)*)?: .+$'
+          error: 'Wrong [ci] commit message'
+          excludeDescription: 'true'
+          excludeTitle: 'true'
+          checkAllCommitMessages: 'true'
+          accessToken: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: None of the above
+        if: steps.feature-match.outcome == 'failure' && steps.refactor-match.outcome == 'failure' && steps.fix-match.outcome == 'failure' && steps.docs-match.outcome == 'failure' && steps.ci-match.outcome == 'failure'
+        run: exit 1
```

Comment:
Since `gsactions/commit-message-checker` Action always makes a boolean conclusion for the particular step, `continue-on-error: true` is mandatory n the case if we want to keep all five regex expressions into the different steps.
So, the final conclusion is to fail the job if all five checks were failed and escape with success if any (one) of expressions was matched.